rules/rbe_repo.bzl

```diff
@@ -846,6 +868,7 @@ def rbe_autoconfig(
         bazel_rc_version = bazel_rc_version,
         digest = digest,
         env = env,
+        create_java_configs = create_java_configs,
```

Comment:
can you add a check above (around lines 828-862) that fails if create_java_configs is false and java_home was set

Reply:
done